sudoexec/bremove

```diff
@@ -147,7 +148,28 @@ for jname in ${JLIST}; do
 		fi
 		# auto-promote if we have children
 		if [ ${zfsfeat} -eq 1 ]; then
-			for disk_path in $( cbsdsql storage_media SELECT path FROM media WHERE jname=\"${jname}\" ); do
+			base_vm_disks=$( cbsdsql storage_media SELECT path FROM media WHERE jname=\"${jname}\" and type=\"hdd\" )
+			for vm_item in $( cbsdsql storage_media SELECT jname,path FROM media WHERE jname!=\"${jname}\" and type=\"hdd\" ); do
+				vm=$( echo ${vm_item} | cut -f 1 -d '|' )
```

Comment:
use please fullpath for external command ( result from: which <cmd> )
/usr/bin/cut  instead of cut
/usr/bin/sed instead of sed
/sbin/zfs instead of zfs 
..